triage/2024-04-22.md

```diff
@@ -0,0 +1,106 @@
+# 2025-04-22 Triage Log
+
+Mostly positive week. Most of the improvements come from a revert of a regression from a few weeks ago, but we also get nice wins from re-using Sized  fast-path, coming from Sized hierarchy implementation work.
```

Comment:
Hm.. maybe I should mention removing Nonterminal, too, that's been a long time coming.

Reply:
You could, but I doubt that anyone outside from a few compiler people know that that means :)